polkadot/zombienet-sdk-tests/tests/smoke/runtime_upgrade.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+use std::env;
+use zombienet_sdk::{tx_helper::RuntimeUpgradeOptions, NetworkConfigBuilder};
+
+const BEST_BLOCK_METRIC: &str = "block_height{status=\"best\"}";
+
+#[tokio::test(flavor = "multi_thread")]
+async fn runtime_upgrade_test() -> Result<(), anyhow::Error> {
+	env_logger::init_from_env(
+		env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+	);
+
+	let images = zombienet_sdk::environment::get_images_from_env();
+	let config = NetworkConfigBuilder::new()
+		.with_relaychain(|r| {
+			r.with_chain("rococo-local")
+				.with_default_command("polkadot")
+				.with_default_image(images.polkadot.as_str())
+				.with_node(|node| {
+					node.with_name("latest-release")
+						// used for getting the chain-spec
+						.with_image("parity/polkadot:latest")
```

Comment:
Can we somehow use the exact runtime-image that roco chain is currently using, or at least the last stable release version, since the upgrades happened all at once.

Reply:
`parity/polkadot:latest` this tag should point to the latest release (stable). 
<img width="938" alt="image" src="https://github.com/user-attachments/assets/d7907bd9-bcbe-4862-b9e5-79f55393782f">

Thx!